senza/components/taupage_auto_scaling_group.py

```diff
@@ -1,37 +1,51 @@
+import json
+import re
+import sys
+import textwrap
 
 import click
 import pierone.api
-import textwrap
 import yaml
-import json
-import sys
-import re
-
 from senza.aws import resolve_referenced_resource
 from senza.components.auto_scaling_group import component_auto_scaling_group
 from senza.docker import docker_image_exists
 from senza.utils import ensure_keys
 
-
 _AWS_FN_RE = re.compile(r"('[{]{2} (.*?) [}]{2}')", re.DOTALL)
 
 
 def check_docker_image_exists(docker_image: pierone.api.DockerImage):
     if 'pierone' in docker_image.registry:
         try:
             exists = pierone.api.image_exists('pierone', docker_image)
+            if exists:
+                image_tag = pierone.api.get_image_tag('pierone', docker_image)
         except pierone.api.Unauthorized:
             msg = textwrap.dedent('''
             Unauthorized: Cannot check whether Docker image "{}" exists in Pier One Docker registry.
             Please generate a "pierone" OAuth access token using "pierone login".
             Alternatively you can skip this check using the "--force" option.
             '''.format(docker_image)).strip()
             raise click.UsageError(msg)
-
     else:
         exists = docker_image_exists(str(docker_image))
+
+    warn_msg = 'You are deploying an image that was not automatically '
+    'checked for vulnerabilities. Images stored in Pierone are automatically checked!'
+
     if not exists:
         raise click.UsageError('Docker image "{}" does not exist'.format(docker_image))
+    elif 'severity_fix_available' in image_tag:
```

Comment:
:scream: You're right! I am changing 4 projects at same time... I need to focus better.